src/styles/blocks/product/card.css

```diff
@@ -0,0 +1,28 @@
+.card {
+  box-sizing: border-box;
+
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #e5e5e5;
+  border-radius: 5px;
+}
+
+.card__image-wrapper {
+  margin-bottom: 40px;
+}
+
+.card__product-title {
+  margin-bottom: 4px;
+}
+
+.card__product-code {
+  margin-bottom: 16px;
+}
+
+.card__rating-wrapper {
+  margin-bottom: 24px;
+}
+
+.card__price-wrapper {
+  margin-bottom: 16px;
+}
```

Comment:
these ones are block elements - no need for creating additional classes for external styles